src/main/java/io/jenkins/infra/repository_permissions_updater/hosting/HostingChecker.java

```diff
@@ -27,7 +27,7 @@ public class HostingChecker {
 
     public static final String INVALID_FORK_FROM = "Repository URL '%s' is not a valid GitHub repository (check that you do not have .git at the end, GitHub API doesn't support this).";
 
-    public static final Version LOWEST_JENKINS_VERSION = new Version(2, 440, 3);
+    public static final Version LOWEST_JENKINS_VERSION = new Version(2, 452, 4);
```

Comment:
Should we go here to 2.462.3 or even 2.479.3?

Reply:
Based on https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/#changing-the-minimum-required-version: `2.479.3`. 

I think there is a automatic update of that value when the documentation is built. Maybe we should use that.

Reply:
That page states:
```
At the moment, the Jenkins releases 2.462.3 and 2.479.3 make good core dependencies. You could also consider 2.492.1 if there are specific reasons, like new features, to want something newer.
```
But the transition to Java 17 I think might justify to go to 2.479.3 now, given that many existing plugins are also updated to use 2.479.x

Reply:
I think so lets go with 2.479.3